demos/simple_jump_bot/README.md

```diff
@@ -0,0 +1,22 @@
+# HammettSimpleJumpBot
+
+This demo represents the `jump` mechanism which Hammett provides. This mechanism allows sending yours screens as new message and saving the previous ones in a chat history.
```

Comment:
This demo showcases the `move` and `jump` features provided by Hammett. `move` allows you to navigate between different screens while re-rendering the same message, and `jump` lets you send screens as a new message, saving the previous ones in the chat history.